doc/generate-casts/replay-patches-multi-demo.sh

```
@@ -0,0 +1,80 @@
+KEYSTROKES=$(cat <<'KEYSTROKES'
+WAIT "Ctrl-C abort" SEND RIGHT DELAY 0.9
+WAIT "\[all/1 patches applied\]" SEND DOWN DELAY 0.9
+WAIT "> jsmn" SEND RIGHT DELAY 0.9
+WAIT "\[all/1 patches applied\]" SEND ENTER DELAY 0.9
+KEYSTROKES
+)
+
+p "dfetch replay-patches --interactive cpputest jsmn"
+echo "$KEYSTROKES" | python3 ../interactive_helper.py replay-patches --interactive cpputest jsmn
+
+PROMPT_TIMEOUT=3
```

Shellcheck (0.11.0)

[warning] PROMPT_TIMEOUT appears unused. Verify use (or export if used externally).

(SC2034)